cmd/livepeer/livepeer.go

```diff
@@ -66,6 +66,7 @@ var (
 const RtmpPort = "1935"
 const RpcPort = "8935"
 const CliPort = "7935"
+const gRPCPort = "6935"
```

Comment:
Do we need to define a new port for this service? Can we reuse the existing `RpcPort` on 8935? In general, we have tried to minimize the number of ports we have exposed, because that directly leads to additional operational complexity.

Reply:
I felt like we could reserve a separate port for multi-purpose services in multi-o setups, eg a blockchain event watching service. 

It thought might be good to separate that traffic from discovery/transcode requests.

Reply:
For standalone redeemers, the discovery / transcode paths would not be enabled. I have a preference to not allow RPC redeemers alongside video stuff, but even if we did mix them up, that seems OK.

More sophisticated operators will either be segregating their redeemer / orchestrator appropriately, rendering the issue moot, or filtering routes via proxy, so they can shape their setups as needed.

Minimizing port usage also minimizes bikeshedding on our part over determining which services are appropriate for a new RPC, or trying to figure out whether a new service justifies a new port. Also see earlier discussion on the topic: https://github.com/livepeer/go-livepeer/issues/781#issuecomment-470718308